config/database.yml

```diff
@@ -1,20 +1,26 @@
-default: &default
+local_default: &local_default
+  adapter: postgresql
+  encoding: unicode
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+
+deploy_default: &deploy_default
```

Comment:
how are deploy_default and local_default different? and is it a problem that we seem to have lost `url: <%= ENV["DATABASE_URL"] %>`?

Reply:
Oh! Thanks for catching this! Meant to make deploy default point to Database URL. I'll push an update.